tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -125,3 +131,106 @@ fn run_ledger_with_ethereum_events_endpoint() -> Result<()> {
 
     Ok(())
 }
+
+/// In this test, we check the following:
+/// 1. We can successfully add tranfers to the bridge pool.
+/// 2. We can query the bridge pool and it is non-empty.
+#[test]
+fn test_add_to_bridge_pool() {
+    const LEDGER_STARTUP_TIMEOUT_SECONDS: u64 = 40;
+    const CLIENT_COMMAND_TIMEOUT_SECONDS: u64 = 60;
+    const QUERY_TIMEOUT_SECONDS: u64 = 40;
+    const SOLE_VALIDATOR: Who = Who::Validator(0);
+    let wnam_address = wnam().to_canonical();
+    let test = setup::network(
+        |mut genesis| {
+            genesis.ethereum_bridge_params = Some(EthereumBridgeConfig {
+                min_confirmations: Default::default(),
+                contracts: Contracts {
+                    native_erc20: wnam(),
+                    bridge: UpgradeableContract {
+                        address: EthAddress([0; 20]),
+                        version: Default::default(),
+                    },
+                    governance: UpgradeableContract {
+                        address: EthAddress([1; 20]),
+                        version: Default::default(),
+                    },
+                },
+            });
+            genesis
+        },
+        None,
+    )
+    .unwrap();
+    set_ethereum_bridge_mode(
+        &test,
+        &test.net.chain_id,
+        &Who::Validator(0),
+        ethereum_bridge::ledger::Mode::EventsEndpoint,
+    );
+
+    let mut namadan_ledger = run_as!(
+        test,
+        SOLE_VALIDATOR,
+        Bin::Node,
+        &["ledger"],
+        Some(LEDGER_STARTUP_TIMEOUT_SECONDS)
+    )
+    .unwrap();
+    namadan_ledger
+        .exp_string("Anoma ledger node started")
+        .unwrap();
+    namadan_ledger
+        .exp_string("Tendermint node started")
+        .unwrap();
+    namadan_ledger.exp_string("Committed block hash").unwrap();
+    let _bg_ledger = namadan_ledger.background();
+
+    let ledger_addr = get_actor_rpc(&test, &SOLE_VALIDATOR);
+    let tx_args = vec![
+        "add-erc20-transfer",
+        "--address",
+        BERTHA,
+        "--signer",
+        BERTHA,
+        "--amount",
+        "100",
+        "--erc20",
+        &wnam_address,
+        "--ethereum-address",
+        DAI_ERC20_ETH_ADDRESS_CHECKSUMMED,
+        "--fee-amount",
+        "10",
+        "--fee-payer",
+        BERTHA,
+        "--gas-amount",
+        "0",
+        "--gas-limit",
+        "0",
+        "--gas-token",
+        NAM,
+        "--ledger-address",
+        &ledger_addr,
+    ];
+
+    let mut namadac_tx = run!(
+        test,
+        Bin::Client,
+        tx_args,
+        Some(CLIENT_COMMAND_TIMEOUT_SECONDS)
+    )
+    .unwrap();
+    namadac_tx.exp_string("Transaction applied").unwrap();
```

Comment:
```suggestion
    namadac_tx.exp_string("Transaction accepted").unwrap();
    namadac_tx.exp_string("Transaction applied").unwrap();
```
should we perhaps add an accepted check?

Reply:
also check if the RPC server is running beforehand